packages/next/src/build/collect-build-traces.ts

```diff
@@ -406,11 +406,9 @@ export async function collectBuildTraces({
               return await fs.readFile(p, 'utf8')
             } catch (e) {
               if (isError(e) && (e.code === 'ENOENT' || e.code === 'EISDIR')) {
-                // handle temporary internal webpack files
                 if (p.match(/static[/\\]media/)) {
                   return ''
                 }
-                return null
```

Comment:
Did you intend to change this line?

Reply:
Yeah attempting to fix flake with ENOENT can move to a separate PR

Reply:
Moved to https://github.com/vercel/next.js/pull/57092